ta/pkcs11/src/pkcs11_token.h

```diff
@@ -153,8 +153,8 @@ struct active_processing {
  *
  * @attributes - matching attributes list searched (null if no search)
  * @count - number of matching handle found
- * @handles - array of handle of matching objects (published handles)
- * @next - index of the next object handle to return to FindObject
+ * @handles - array of handle of matching objects
+ * @next - index of the next object handle to return to C_FindObject
```

Comment:
very minor: this change in commit "ta: pkcs11: Add support for getting object size and attribute value" should move into the previous commit "ta: pkcs11: Add support for finding objects".